src/Combobox/index.js

```diff
@@ -17,14 +18,26 @@ export const VALID_ICON_NAMES = iconSelection.icons.map(
   (icon) => icon.properties.name,
 );
 
+/**
+ * @param {Object} item Combobox.{Action|Item|Heading} component
+ * @returns {Boolean} true if the item is a Combobox.Action
+ */
+export const isAction = (item) => {
```

Comment:
Oh wow, I'm behind the times: was initially thinking that this would be more literally an isAction function if we could enforce ComboboxAction.propTypes on item... but we can't do that anymore and they raise "Invariant Violation: Calling PropTypes validators directly is not supported by the `prop-types` package." and don't allow a way of calling PropTypes.checkPropTypes() in a way that returns or raises errors =(

I suppose we could call this getLabel() or hasLabel(), but so long as we don't add too many other Combobox elements it shouldn't pose too much of a problem

Reply:
Yeah, I'm not crazy about this as it is. I'm making my way through components converting things to typescript, which should make some of these checks and validation much better